kong-ingress-data-plane-postgres.yaml

```diff
@@ -75,5 +79,13 @@ spec:
     port: 443
     targetPort: 8443
     protocol: TCP
+  - name: http
+    port: 80
+    targetPort: 8001
+    protocol: TCP
+  - name: https
+    port: 443
```

Comment:
Same here for 443.

Reply:
agreed fixed on the parent branch f0d9f53be25e49a1c1cef1b84eb5b1a9014e137a